CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Lotus changelog
 
 # UNRELEASED
+ - We're shipping a new Indexer implementation in Lotus (`ChainIndexer`) to index Filecoin chain state such as tipsets, messages, events and ETH transactions for reliable and faster RPC responses. The `ChainIndexer` replaces the existing `MsgIndex`, `EthTxIndex` and `EventIndex` implementations in Lotus which suffer from a multitude of known problems documented [here](https://github.com/filecoin-project/lotus/issues/12293). If you are an RPC provider/node operator who serves RPC requests, please refer to the [ChainIndexer documentation for RPC providers](TODO: URL) on the Lotus docs website for information on how to enable, configure and use the new Indexer.
```

Comment:
@BigLep @rjan90 Any comments ?

Reply:
Keeping this comment unresolved till we have a URL for the Lotus docs.

Reply:
```suggestion
 (`ChainIndexer`) to index Filecoin chain state such as tipsets, messages, events and ETH transactions for accurate and faster RPC responses. The `ChainIndexer` replaces the existing `MsgIndex`, `EthTxIndex` and `EventIndex` implementations in Lotus which[ suffer from a multitude of known problems](https://github.com/filecoin-project/lotus/issues/12293).  If you are an RPC provider/node operator , please refer to the [ChainIndexer documentation for RPC providers](TODO: URL) for information on how to enable, configure and use the new Indexer.
```

Since this is in the changelog, it's understood that it has shipped.  
Made it more concise.
I went with accurate over reliable to emphasize that a big win here is correctness.  We're also moving to being correct - no gaps/missing data.  That is why I'm saying "accurate" instead of "more accurate". Being correct is tablestakes.  I don't want to leave the impression that there are still correctness issues.